lang/cs/Org.Apache.REEF.IMRU/Org.Apache.REEF.IMRU.csproj

```diff
@@ -74,6 +76,9 @@ under the License.
     <Compile Include="OnREEF\Driver\DataLoadingContext.cs" />
     <Compile Include="OnREEF\Driver\EvaluatorManager.cs" />
     <Compile Include="OnREEF\Driver\EvaluatorSpecification.cs" />
+    <Compile Include="OnREEF\Driver\IJobLifeCycleManager.cs" />
```

Comment:
Do we expect the client to implement IJobLifeCycleManager? If yes, it should be under API folder, otherwise it should be internal.